### ISSUE TITLE
SRS plugin prevents me opening DCS's original PTT window which is necessary for fixing a know DCS Syria map problem 

### ISSUE THREAD

**Issue by XtroTheArctic** [ciribob/DCS-SimpleRadioStandalone#734]
Hello.

As some of you guys may know, there is a radio related problem in DCS syria map. Here is the details of the problem and how to fix it:
https://forum.dcs.world/topic/339355-f-15c-has-loud-static-noise-in-some-regions-of-the-map-since-recent-updates/
You can scroll SparrowLT's post starting with "Did more testing in multiplayer in Syria" to see the solution.

To turn the volume of radio 2 down, or to change the frequency of radio 2, we need to open DCS's original radio window.

I can open that window when I am connected to a Syria server which doesn't have SRS advertisement and turn the volume of radio 2 to get rid of the noise.

When I'm connected to a server which has SRS advertisement, SRS plugin doesn't allow me to open DCS's radio window and I get stuck with this static noise.

I tried setting up radio 2 key bindings to turn the volume down or changing the frequency without needing to open the radio window but still SRS plugin doesn't allow those key bindings to work properly.

I am in need of urgent help. Please provide a solution.

**Comment by ciribob** [comment]
SRS doesn't block the DCS VoIP system - they can be used in parallel so i'm going to close this issue for now as I dont follow. You also dont list the aircraft as different aircraft have different SRS binds

**Comment by XtroTheArctic** [comment]
Hello again.

Sorry I didn't mention the aircraft but it was mentioned in the link I provided already. It's very well reported that F-15c is picking up a constant static noise in Syria map on Radio 2. Of-course this is a DCS issue, not an SRS issue but... as I reported in this ticket, SRS somehow blocks opening DCS's radio controls window ONLY WHEN the connected multiplayer server advertises SRS channels. When I connect to a Syria server without SRS advertisement, I can open the radio controls window and silence radio 2 to get rid of the static.

Since I can open the radio controls windows on SRS-less multiplayer server, I can confirm that my aircraft (F-15c) keybinding for opening the radio controls window is working fine. Only the SRS should be blocking it when the server is advertising SRS channels which somehow activates some SRS functions which somehow deactivates the key bindings for DCS's radio controls window. 

I may have been referring to the radio controls window with a wrong name thou. It appears as "Show VoiceChat window" in control options window. Also, it's in VoiceChat category, not in F-15c (aircraft) category.

As I mentioned before, this key binding stops working only when I'm connected to a multiplayer server which advertises SRS connection info.

![image](https://github.com/user-attachments/assets/fcb4b952-bfb9-4d62-92f2-a00aed208895)
